pkg/finder/noderesource-internal_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package finder
+
+
+import (
+  "reflect"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+  	"k8s.io/api/core/v1"
+  podresourcesapi "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+)
+
+
+func TestGetNodeCPUInfo(t *testing.T) {
+	Convey("Testing getNodeCPUInfo ", t, func() {
+		Convey("Sysfs specified", func() {
+      cpuIds, _ := getList("/sys/devices/system/cpu/online")
+      expectedNumanodes, err := getList("/sys/devices/system/node/online")
+			numanodes, NUMANode2CPUs, cpu2NUMA, err := getNodeCPUInfo("/sys")
```

Comment:
Well, is hard to unit test when we are using the system data. We should create some files and read from them so we can capture different systems... or am I missing something?

Reply:
Yep, that sounds right, I should be using test files here instead. Will update to that. Certainly refactoring could help here